test/integration/messaging.js

```diff
@@ -27,25 +27,29 @@ describe('messaging', function() {
   });
 
   it('updates a message', function(done) {
-    senderModel.update({topic: 'public', message: 'An update message'},
-    function(err) {
-      if (err) return done(err);
-      receiverModel.find(0, function(msg) {
-        expect(msg).to.equal('An update message');
-        done();
+    setTimeout(function() {
+      senderModel.update({topic: 'public', message: 'An update message'},
+      function(err) {
+        if (err) return done(err);
+        receiverModel.find(0, function(msg) {
+          expect(msg).to.equal('An update message');
+          done();
+        });
       });
-    });
+    }, 5000);
```

Comment:
can we add a timeout variable on top of the file and just call it on the needed test cases?